clamav/1.0/alpine/Dockerfile

```diff
@@ -35,6 +35,7 @@ RUN apk update && apk upgrade \
         # For building static libraries with Mussels
         git \
         patchelf \
+        shadow \
```

Comment:
It doesn't seem like this needs to be added to the first image (the "builder" image) as the entrypoint script is based on the second image.